lib/Migration/Version2000Date20220104144742.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Mail\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version2000Date20220104144742 extends SimpleMigrationStep {
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		$schema = $schemaClosure();
+
+		$recipientsTable = $schema->getTable('mail_recipients');
+		$recipientsTable->addColumn('mailbox_type', 'integer', [
+			'notnull' => true,
+			'default' => 0,
+		]);
+
+		$localMailboxTable = $schema->createTable('mail_local_mailbox');
+		$localMailboxTable->addColumn('id', 'integer', [
+			'autoincrement' => true,
+			'notnull' => true,
+			'length' => 4,
+		]);
+		$localMailboxTable->addColumn('type', 'integer', [
+			'notnull' => true,
+			'unsigned' => true,
+			'length' => 1,
+		]);
+		$localMailboxTable->addColumn('account_id', 'integer', [
+			'notnull' => true,
+			'length' => 4,
+		]);
+		$localMailboxTable->addColumn('alias_id', 'integer', [
+			'notnull' => false,
+			'length' => 4,
+		]);
+		$localMailboxTable->addColumn('send_at', 'integer', [
+			'notnull' => false,
+			'length' => 4
+		]);
+		$localMailboxTable->addColumn('subject', 'text', [
+			'notnull' => true,
+			'length' => 255
+		]);
+		$localMailboxTable->addColumn('body', 'text', [
+			'notnull' => true
+		]);
+		$localMailboxTable->addColumn('html', 'boolean', [
+			'notnull' => false,
+			'default' => false,
+		]);
+		$localMailboxTable->addColumn('in_reply_to_id', 'integer', [
+			'notnull' => false,
+			'length' => 4,
+		]);
+		$localMailboxTable->addColumn('draft_id', 'integer', [
+			'notnull' => false,
+			'length' => 4,
+		]);
+		$localMailboxTable->setPrimaryKey(['id']);
+
+		$attachmentsTable = $schema->createTable('mail_lcl_mbx_attchmts');
```

Comment:
isn't `mail_local_mb_attachments` still short enough?

Reply:
![image](https://user-images.githubusercontent.com/7427347/154048333-6b8cc364-b31f-48dc-9f1e-525c28d5de84.png)

no sorry :( but mb instead of mbx is nicer

Reply:
Compormise on `mail_local_mb_attchmts` ? :grin: